### DISCUSSION TITLE
Changed soremote password, sensors still work

### DISCUSSION THREAD

**Discussion by stijnpieters** [Security-Onion-Solutions/securityonion#1775] [Unsupported Versions] [answered]
I changed the password of the soremote user on my manager node, do I need to change anything on my sensors? 

Currently the sensors still send data to the manager, but I want to be sure they don't suddenly stop working.
I am unfamiliar with saltstack, thus I don't know if this password is still required if the minion has already connected to its master.

**Comment by TOoSmOotH** [comment, marked as answer]
The only time soremote is used is when you are adding new nodes. You can disable this account until you need to add a new sensor/search node etc. SO2 does not use SSH tunnels.
